src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -628,7 +629,7 @@ namespace AppInstaller::CLI::Workflow
 
         for (size_t i = 0; i < searchResult.Matches.size(); ++i)
         {
-            auto latestVersion = searchResult.Matches[i].Package->GetLatestAvailableVersion();
+            auto latestVersion = GetAvailableVersionsForInstalledVersion(searchResult.Matches[i].Package, nullptr)->GetLatestVersion();

```

Comment:
Could we add a `GetAvailableVersions(ICompositePackage)` so that we don't have to pass `nullptr`?